rpc/src/crab.rs

```diff
@@ -243,38 +228,6 @@ where
 		true,
 	)));
 	io.extend_with(Web3ApiServer::to_delegate(Web3Api::new(client)));
-	// --- dvm --->
```

Comment:
```rs
let ethapi_cmd = rpc_config.ethapi.clone();

if ethapi_cmd.contains(&EthApiCmd::Debug) || ethapi_cmd.contains(&EthApiCmd::Trace) {
	if let Some(trace_filter_requester) = tracing_requesters.trace {
		io.extend_with(TraceApiServer::to_delegate(Trace::new(
			client,
			trace_filter_requester,
			rpc_config.ethapi_trace_max_count,
		)));
	}

	if let Some(debug_requester) = tracing_requesters.debug {
		io.extend_with(DebugApiServer::to_delegate(Debug::new(debug_requester)));
	}
}
```

Reply:
As for `evm-tracing` feature, it would be better to add evm-tracing companion like others and then disable it before merging.